minimesos/src/main/java/com/containersol/minimesos/state/State.java

```diff
@@ -23,6 +26,13 @@ public static State fromJSON(String jsonString) throws JsonParseException, JsonM
         }
     }
 
+    private Map<String, String> flags = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1213)